app/src/main/java/org/oppia/app/profile/ProfileChooserFragmentPresenter.kt

```diff
@@ -59,6 +65,12 @@ class ProfileChooserFragmentPresenter @Inject constructor(
   private val chooserViewModel: ProfileChooserViewModel by lazy {
     getProfileChooserViewModel()
   }
+  private val profileLiveData: LiveData<AsyncResult<Boolean>> by lazy {
```

Comment:
Rename this to `wasProfileEverCreatedLiveData`

Reply:
Done